src/extension.ts

```diff
@@ -36,7 +36,7 @@ export async function activate(context: ExtensionContext): Promise<CodeForIBMi>
 
   await loadAllofExtension(context);
   const checkLastConnections = () => {
-    const connections = (GlobalConfiguration.get<ConnectionData[]>(`connections`) || []);
+    const connections = (ConnectionManager.getAll() || []);
```

Comment:
`getAll` always returns an array.
```suggestion
    const connections = ConnectionManager.getAll();
```